addon/lib/toolbar-button.js

```diff
@@ -68,9 +69,22 @@ function getExperimentList(availableExperiments, installedAddons) {
     }
     return experiment;
   })
-  .filter(x => x.active || !x.completed); // remove inactive, completed experiments
+  // Remove inactive, completed experiments.
+  .filter(x => x.active || !x.completed)
+  // Remove experiments with a locale_grantlist, unless the user's primary
+  // locale is in the grantlist. #1961
+  .filter(x => {
+    if (!('locale_grantlist' in x)) {
+      return true;
+    }
+
+    // In keeping with the approach of #1602, we ignore the region in the
+    // user's locale.
+    const userLocale = Services.appShell.hiddenDOMWindow.navigator.language.split('-')[0];
+    return x.locale_grantlist.includes(userLocale);
+  });
```

Comment:
Do we need to check the `locale_blocklist` as well? [/docs/content/template.yaml:94-99](https://github.com/mozilla/testpilot/blob/97fdd0735024f85014aa8645ef0dc763a8aadcf7/docs/content/template.yaml#L94-L99)

([Ref](https://github.com/mozilla/testpilot/blob/2218655b4ecc33a53dfc42d2cc4de91d3a45047e/docs/content/reference.md#locales))